test/tst_histogram.jl

```diff
@@ -1,5 +1,6 @@
 seed!(RNG, 1337)
 x = randn(RNG, 10000)
+x2 = randn(RNG, (1, 10000, 1, 1))
```

Comment:
```suggestion
x2 = reshape(x, (1, 10000, 1, 1))
```

Reply:
Ha, yeah, even better, reshape in the test.